src/stratigraphy/util/predictions.py

```diff
@@ -274,6 +286,43 @@ def evaluate(self, ground_truth_layers: list):
                 layer.material_is_correct = False
                 layer.depth_interval_is_correct = None
 
+    def evaluate_metadata(self, metadata_ground_truth: dict):
+        """Evaluate the metadata of the file against the ground truth.
+
+        Note: For now coordinates is the only metadata extracted and evaluated for.
+
+        Args:
+            metadata_ground_truth (dict): The ground truth for the file.
+        """
+        if self.metadata.coordinates is None:
+            self.metadata_is_correct["coordinates"] = False
```

Comment:
Maybe in future we should relax this a little bit, and say that it's ok when no coordinates are extracted, as long as there are no coordinates in the ground truth either.
Currently this will not make a difference, as there are always coordinates in the ground truth. However, then are not always contained in the PDFs. So maybe in future, we will want to set the ground truth to None when the coordinates cannot be extracted from the PDF at all.